src/player.py

```diff
@@ -50,18 +51,16 @@ def __init__(
         self.velocity = (0.0, 0.0)
         "Translational velocity -- (x,y) tuple -- measured in pixels per second"
 
+        self.vehicle = MovementControls(
+            LinkedSprite[Player](texture=RED_CAR, scale=0.18)
+        )
+
     def update(self, delta_time: float):
         #
         # Driving and movement
         #
-        if self.input.debug_3.pressed:
```

Comment:
does this mean we are done testing various driving modes? Looks like removing the ability to swap driving modes